re/schedule.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package re
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type Schedule struct {
+	StartDateTime   time.Time     `json:"start_datetime"`          // When the schedule becomes active
+	RecurringTime   time.Time     `json:"recurring_time"`          // Specific time for the rule to run
+	RecurringType   ReccuringType `json:"recurring_type"`          // None, Daily, Weekly, Monthly
+	RecurringPeriod uint          `json:"recurring_period"`        // Controls how many intervals to skip between executions: 1 runs at every interval, 2 runs at every second interval, etc.
```

Comment:
Controls how many intervals to skip between executions: 1 = every Recurring value, 2 = every other Recurring value...